services/transcode/subtitles_test.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func Test_SubtitlesBurnIn(t *testing.T) {
+	t.Skip("TODO: setup test data")
```

Comment:
forgotten or on purpose?

Reply:
This one is on purpose, because I estimated ~2-4h to make the test be of real value (need to write the video generators), but did not want to spend that time right now.